java/hsfs/src/main/java/com/logicalclocks/hsfs/engine/VectorServer.java

```diff
@@ -383,7 +384,7 @@ private void setupJdbcConnection(FeatureStoreBase featureStoreBase, Boolean exte
       // if external is true, replace the IP coming from the storage connector with the host
       // used during the connection setup
       String host;
-      Optional<Variable> loadbalancerVariable = variablesApi.get(VariablesApi.LOADBALANCER_EXTERNAL_DOMAIN);
+      Optional<Variable> loadbalancerVariable = variablesApi.get(VariablesApi.LOADBALANCER_EXTERNAL_DOMAIN_MYSQL);
```

Comment:
I guess the fallback here that assumes mysql is running on the head node is not correct anymore in the kubernetes world since we don't have a head node abstraction and services are in pods that may or may not run on a given host. Not sure what the correct fallback is here.

Reply:
There is no fallback. I will raise an exception for it.